README.md

```diff
@@ -0,0 +1,31 @@
+![PyTest](https://github.com/SwarmRL/SwarmRL/blob/master/.github/workflows/pytest.yml/badge.svg)
+[![code-style](https://img.shields.io/badge/code%20style-black-black)](https://github.com/psf/black/)
+
+# SwarmRL
+SwarmRL is a Python package aimed at providing a simple and flexible framework for
+developing and testing reinforcement learning and mathematical optimization algorithms
+for multi-agent systems.
+Some key features of SwarmRL include:
+
+* Integration with EspressoMD simulation engine
+* Support for experimental setups
+* Actor critic reinforcement learning strategies
+
+## Installation
+
+Currently SwamrRL is only available from source so it must be installed within the local
+directory.
+
+```sh
+   git clone https://github.com/SwarmRL/SwarmRL.git
+   cd SwarmRL
+   pip install .
+```
+
+This will install the package with a reference to the current directory.
+This means that if you change the code, you will not need to re-install it.
```

Comment:
the pip command misses ``-e``. Also not sure if we need that level of installation instructions in our readme

Reply:
I can just have the pip install line and say this will install it but without the discussion local installation?